blockchain/v2/reactor.go

```diff
@@ -59,7 +59,11 @@ type blockApplier interface {
 // XXX: unify naming in this package around tmState
 func newReactor(state state.State, store blockStore, reporter behaviour.Reporter,
 	blockApplier blockApplier, fastSync bool) *BlockchainReactor {
-	scheduler := newScheduler(state.LastBlockHeight, time.Now())
+	initHeight := state.LastBlockHeight + 1
```

Comment:
does it make sense to extract this into a func

```go
func (s State) InitHeight() int64 {
  	initHeight := s.LastBlockHeight + 1
	if initHeight == 1 {
		return s.InitialHeight
	}
        return initHeight
}
```

Reply:
Yeah, that might simplify things. It would have to be `NextHeight()` or something, probably with a boolean indicating whether it's the initial block, but I'll give it a try.